### PR TITLE
Jude/chore/refactor locations api

### PR DESCRIPTION
**Pull request by judeokafor**

### CODE REVIEW

File: functions/src/controllers/commercials.js [hunk 1]
```diff
@@ -28,12 +28,13 @@ class commercialController {
             const duration = await getVideoDurationInSeconds(filePath);
             const { url, id } = await uploads(filePath);
             const newCommercial = {
-                createdAt: new Date().toISOString(),
-                createdBy: userId, description, duration: Math.round(duration), title,
-                url, videoId: id,
+                createdAt: new Date().toISOString(), createdBy: userId, description,
+                duration: Math.round(duration), title, url, videoId: id,
```
Comment (Way2nnadi) [inline functions/src/controllers/commercials.js]: put the keys on their own line

**Review by Way2nnadi — APPROVED** [review]
minor change

**Review by Simpcyclassy — APPROVED** [review]
Looks good to me

File: functions/src/controllers/commercials.js [hunk 2]
```diff
@@ -47,15 +45,17 @@ class commercialController {
...
-            for (const doc of docs) {
-                const newObj = Object.assign({}, doc.data(), { commercialId: doc.id });
-                commercials.push(newObj);
-            }
+            const commercialData = docs.map(async doc => {
+                const commercialResponseData = createCommercialResponseData(doc);
+                return commercialResponseData;
+            }); const commercials = await Promise.all(commercialData);
+            console.log(commercials);
             return successNoMessage(res, OK, commercials);
```
Comment (Simpcyclassy) [inline functions/src/controllers/commercials.js:59]: Please remove console logging

**Pull request merged**